docker-containers.el

```diff
@@ -284,12 +283,16 @@ Remove the volumes associated with the container when VOLUMES is set."
 container selected run it in compile-mode."
   (interactive)
   (let* ((id-list (docker-utils-get-marked-items-ids))
-         (containers (mapconcat 'identity id-list " "))
-         (docker-logs-command (format "docker logs %s" containers)))
-    (if (= 1 (length id-list))
-        (docker-logs-show (car id-list))
-      (docker-containers-run-command-on-selection-print "logs"
-                                                        (s-join " " (docker-containers-logs-arguments))))))
+         (args (docker-containers-logs-arguments))
+         (singlep (= 1 (length id-list)))
+         (followp (-contains? args "-f")))
+    (if (and singlep followp)
+        (let* ((docker-logs-command (format "docker logs -f %s" (car id-list)))
+               (docker-logs-buffer (get-buffer-create (concat "*" docker-logs-command "*"))))
+          (async-shell-command docker-logs-command docker-logs-buffer))
+      (if followp
+          (message "-f is only supported for one container")
+        (docker-containers-run-command-on-selection-print "logs" "")))))
 
```

Comment:
I think it makes more sense to always use `async-shell-command`, for one or more containers in a loop.

Reply:
Great idea! Done!